sympy/assumptions/relation/equality.py

```diff
@@ -1,67 +1,59 @@
 """
-Module for mathematical equality [1] and inequation [2].
+Module for mathematical equality [1] and inequalities [2].
+
+The purpose of this module is to provide the instances which represent the
+binary predicates in order to combine the relationals into logical inference
+system. Every class in this module should remain internal to assumptions module,
+and user must use the classes in ``core/relational`` instead to construct the
+relational expressions.
```

Comment:
This would be clearer with examples of which classes are internal and which should be used